install_config/persistent_storage/topics/glusterfs_example_basic.adoc

```diff
@@ -1,5 +1,20 @@
-. In your inventory file, add `glusterfs` in the `[OSEv3:children]` section to
-enable the `[glusterfs]` group:
+. In your inventory file, include the following variables in the `[OSEv3:vars]`
+section, adjusting them as needed for your configuration:
```

Comment:
s/section, adjusting them as needed for your configuration/section, and adjust them as required for your configuration.